.github/workflows/build_artifacts.yaml

```diff
@@ -1,20 +1,32 @@
 name: build_artifacts
 on:
-  repository_dispatch:
-    types: [ build ]
   workflow_dispatch:
-    inputs:
-      opendal_core_version:
-        description: "Version of Apache OpenDAL Core"
-        required: true
-        type: string
-      opendal_go_version:
-        description: "Version of Apache OpenDAL Go Binding"
-        required: false
-        type: string
+  schedule:
+    - cron: '*/5 * * * *'
+
 jobs:
+  version_check:
+    runs-on: ubuntu-latest
+    outputs:
+      opendal_core_version: ${{ steps.set-version.outputs.opendal_core_version }}
+      opendal_go_version: ${{ steps.set-version.outputs.opendal_go_version }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+      - name: Setup Version
```

Comment:
Although it's fine to use Python here, could we directly use [actions/github-script](https://github.com/actions/github-script) to avoid introducing extra dependencies?

Reply:
LGTM